tester/orb.yaml

```diff
@@ -162,11 +163,28 @@ jobs:
         description: >
           Docker-compose args such as a chain of files to use.
         type: string
+      creds:
```

Comment:
We should probably label both this and `project` with a `gcr_` prefix, at least, to make clear what they refer to!

Reply:
Done!